### ISSUE TITLE
Possible to change package name?

### ISSUE THREAD

**Issue by will-stone** [Binaryify/OneDark-Pro#54]
The current package name is `"name": "Material-theme"`. Is it possible to change this? Or will it screw-up users' installation? It may have been originally forked from the Material Theme but the name doesn't really go with the project now.

**Comment by Binaryify** [comment]
You can see this issues https://github.com/Microsoft/vscode/issues/25988

**Comment by will-stone** [comment]
Ah ok. If you're unable to change it, you may have to bite-the-bullet and change the name and lose the downloads. I don't think an Atom One Dark theme should have any references to Material, it'll mislead or confuse users.

**Comment by Binaryify** [comment]
I think it is difficult

**Comment by Binaryify** [comment]
At fitst, I like the name "Mateial theme", but this issue https://github.com/Binaryify/OneDark-Pro/issues/40 make me to rename this theme

**Comment by will-stone** [comment]
Indeed. I understand the need for a name change and One Dark Pro is certainly the better choice. However, keeping the package name as Material isn't great. It might be a case of deprecating this and starting a new repo. You could put a big message at the top of the README and a link to the new theme. Could also change logo to small message: "deprecated" too. It's drastic, I know, but will make sense in the long run. That's unless MS can rename it, but I doubt they can.

**Comment by Binaryify** [comment]
@will-stone 
Can you help me to contact the VS Code team to solve this thing?

**Comment by will-stone** [comment]
It looks like this has been assigned a Microsoft representative so we'll have to wait and see what they say. I'm just outlining some options if it's impossible to rename (which it probably is as that ID will be how users' VSCode's know which extension to download). I've never published an extension before, does this page allow you to change anything? https://marketplace.visualstudio.com/manage

**Comment by Binaryify** [comment]
@will-stone 
Seem I can't change package name with no lose the downloads
https://github.com/Microsoft/vscode/issues/25988
But I publish the new theme with new package name
https://marketplace.visualstudio.com/search?term=zhuangtongfa&target=VSCode&category=Themes&sortBy=Relevance

**Comment by ajitid** [comment]
But this theme doesn't get updates

**Comment by Binaryify** [comment]
@ajitid 
Oh, I forget it, now it update

**Comment by will-stone** [comment]
Is the plan to deprecate the old version? I'm not sure what the process would be, just interested.

**Comment by Binaryify** [comment]
@will-stone 
I don't know what to do better

**Comment by jens1o** [comment]
@Binaryify I think with the next update, we should place a notice, so that more and more users are installing the new version.

**Comment by Binaryify** [comment]
@jens1o @will-stone 
I think we can make it be a "Beautiful mistake"

**Comment by jens1o** [comment]
So we're just using the old version, even though it's identifier is wrong?

**Comment by Binaryify** [comment]
@jens1o 
I think that's acceptable

**Comment by jens1o** [comment]
So we can close this issue? Maybe you pull down the new version?

**Comment by RavinduL** [comment]
I accidentally uninstalled this extension upon noticing that the identifier `zhuangtongfa.Material-theme` was listed in the output of the `code --list-extensions` command 🤷‍♂️ .

It is my opinion that, despite the reputation of this extension, it should be published with a new identifier that isn't misleading, as,

* The current identifier is _not at all_ descriptive of the package.
* Material design is a widely known design language.

**Comment by will-stone** [comment]
@Binaryify Looks like my prediction of "it'll mislead or confuse users" has come true 😮

**Comment by Binaryify** [comment]
@will-stone 
still waiting vscode team improve it
https://github.com/Microsoft/vscode/issues/34812#issuecomment-331450423

**Comment by will-stone** [comment]
Awesome. Looks like it's in the works.

**Comment by Binaryify** [comment]
@will-stone 
yes!

**Comment by Binaryify** [comment]
in the insider version can change the extension name or publisher now
![image](https://user-images.githubusercontent.com/12221718/32403224-21dffc4c-c16f-11e7-8cb6-5a447bb2e16e.png)

**Comment by sxyugao** [comment]
Do you know how to change package name now? It let me bother.

**Comment by Huxpro** [comment]
> in the insider version can change the extension name or publisher now
> ![image](https://user-images.githubusercontent.com/12221718/32403224-21dffc4c-c16f-11e7-8cb6-5a447bb2e16e.png)
> but I still don't know how to change it

Did this feature land?

**Comment by ElecTreeFrying** [comment]
@Binaryify have you tried updating`"name"` in package.json to `"one-dark-pro"`? I also wanted to rename my extension. I resolved into what you did I renamed the `"displayName"`.

**Comment by Binaryify** [comment]
@ElecTreeFrying that will lose download and install data
